x-pack/plugins/security_solution/public/management/links.test.ts

```diff
@@ -95,106 +111,101 @@ describe('links', () => {
   });
 
   describe('Host Isolation Exception', () => {
```

Comment:
awesome! thanks a lot for updating these tests!

Reply:
👍 

I just finished merging in your PR and how I did not loose any of your changes (I had merge conflicts 🤞 )